app/views/repositories/_save_repository_filter_modal.html.erb

```diff
@@ -16,8 +16,8 @@
       </div>
 
       <div class="modal-footer">
-        <button type="button" class="btn btn-default" data-dismiss="modal"><%= t('general.cancel')%></button>
-        <button class="btn btn-success" id="saveRepositoryTableFilterButton"><%= t('general.save') %></button>
+        <button title="close filter modal" type="button" class="btn btn-default" data-dismiss="modal"><%= t('general.cancel')%></button>
+        <button title="save filter modal" class="btn btn-success" id="saveRepositoryTableFilterButton"><%= t('general.save') %></button>
```

Comment:
add repository to the above, "close repository filter modal header", "close repository filter modal", ...